probe/endpoint/nat.go

```diff
@@ -60,6 +60,10 @@ func (n natMapper) applyNAT(rpt report.Report, scope string) {
 		if !ok {
 			return
 		}
+		// Don't apply nat mappings to conntracked connections
+		if _, procspied := node.Latest.Lookup(Procspied); !procspied {
```

Comment:
Why not? If we do this we should move SNATing for short-lived connections to report.go